README.md

```diff
@@ -54,9 +54,7 @@ For more installation information see [installing on Windows](/docs/installation
 To install the collector on macOS use the following brew commands:
```

Comment:
Should this be changed to no longer say "brew commands"?

Reply:
good catch